cluster-autoscaler/processors/provreq/provisioning_request_processors_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package provreq
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	apiv1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/autoscaler/cluster-autoscaler/context"
+	. "k8s.io/autoscaler/cluster-autoscaler/utils/test"
+)
+
+func TestProvisioningRequestPodsFilter(t *testing.T) {
```

Comment:
I think I would like to have the event logic covered by the tests as well

Reply:
Added

Reply:
I would add test cases where more events are logged, and also over-riden the limit and check that we log only up to the limit.

Reply:
done

Reply:
Can we also add one or two more cases to this table test in TestProvisioningRequestPodsFilter?

Reply:
Sure, sorry missed that you've asked it earlier

Reply:
Feel free to close this comment